pom.xml

```diff
@@ -744,7 +748,7 @@
             <dependency>
                 <groupId>org.javassist</groupId>
                 <artifactId>javassist</artifactId>
-                <version>3.19.0-GA</version>
+                <version>3.21.0-GA</version>
```

Comment:
Unrelated to your PR, but do you remember what we are using `javassist` for ?

Reply:
It's a transitive dependency (e.g. by Swagger). We declare it to fix some versioning conflicts.